### PR TITLE
Update schedule.md

### PR DESCRIPTION
**Pull request by FuShixiong**
Update Timeline

### What problem does this PR solve?

- Issue number: close #xxx 
- Breif description of the problem: update the schedule of TCP2021

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Side effects

 - Possible performance regression
 - Increased code complexity

Related changes

 - Need to update the documentation

### CODE REVIEW

File: talent-challenge-program2021/schedule.md [hunk 1]
```diff
@@ -5,7 +5,41 @@ _Status: ongoing_
 ### Timeline
 _Note: this timeline is proposed and may be subject to change before the formal program launch announcement_
 
+#### Season I
```
Comment (zz-jason) [inline talent-challenge-program2021/schedule.md:8]:
```suggestion
#### Season 1
```

File: talent-challenge-program2021/schedule.md [hunk 2]
```diff
@@ -5,7 +5,41 @@ _Status: ongoing_
...
+- July 19 - mentees are selected by the mentors, coding starts;
+- 1st evaluation checkpoint (mentor decides);
+    - Mentors verify the quality of the completed tasks;
+    - First 50% of the stipend is being paid to the mentee if this stage is passed;
+- 2nd (final) evaluation checkpoint (mentor decides);
+    - Coding ends;
+    - The other 50% of the stipend is being paid to the mentees;
+    - results announced!
+
+#### Season II
```
Comment (zz-jason) [inline talent-challenge-program2021/schedule.md:20]:
```suggestion
#### Season 2
```

File: talent-challenge-program2021/schedule.md [hunk 3]
```diff
@@ -5,7 +5,41 @@ _Status: ongoing_
...
+- Sep 20 - mentees are selected by the mentors, coding starts;
+- 1st evaluation checkpoint (mentor decides);
+    - Mentors verify the quality of the completed tasks;
+    - First 50% of the stipend is being paid to the mentee if this stage is passed;
+- 2nd (final) evaluation checkpoint (mentor decides);
+    - Coding ends;
+    - The other 50% of the stipend is being paid to the mentees;
+    - results announced!
+
+#### Season III
```
Comment (zz-jason) [inline talent-challenge-program2021/schedule.md:32]:
```suggestion
#### Season 3
```

Comment (FuShixiong) [reply talent-challenge-program2021/schedule.md]: Done

**Review by zz-jason — APPROVED** [review]

**Pull request merged**